sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/Field.java

```diff
@@ -759,4 +771,35 @@ public Field setFields(List<Field> fields) {
         this.fields = fields;
         return this;
     }
+
+    /**
+     * Get the hidden property: A value indicating whether the field will be
+     * returned in a search result. This property must be false for key fields,
+     * and must be null for complex fields. You can hide a field from search
+     * results if you want to use it only as a filter, for sorting, or for
+     * scoring. This property can also be changed on existing fields and
+     * enabling it does not cause an increase in index storage requirements.
+     *
+     * @return the hidden value.
+     */
+    public Boolean isHidden() {
+        return retrievable == null ? null : !retrievable;
+    }
+
+    /**
+     * Set the hidden property: A value indicating whether the field will be
+     * returned in a search result. This property must be true for key fields,
```

Comment:
Need to update this spot as well 😄 

```suggestion
     * returned in a search result. This property must be false for key fields,
```